sycl/include/CL/sycl/handler.hpp

```diff
@@ -504,47 +504,47 @@ class __SYCL_EXPORT handler {
 
   template <typename T, int Dim, access::mode Mode, access::target Target,
             access::placeholder IsPH>
-  detail::enable_if_t<Dim == 0 && Mode == access::mode::atomic, T>
-  readFromFirstAccElement(accessor<T, Dim, Mode, Target, IsPH> Src) const {
+  static detail::enable_if_t<Dim == 0 && Mode == access::mode::atomic, T>
```

Comment:
These functions get called inside SYCL kernels and thus resulting in implicit capture of 'this' pointer. Marking them static to avoid that.

Reply:
I have no objections to make this methods static, they don't access members of the handler.